### PR TITLE
feat: add 'format' option to Page Layout API

### PR DESCRIPTION
**Pull request by samuelmaddock**
Adds a missing option to specify the format returned from the Page Layout API.

### CODE REVIEW

**Comment by fmontes** [comment]
Hi Samuel, thanks for your contribution!

File: projects/dotcms/src/lib/api/DotApiPage.ts [hunk 1]
```diff
@@ -3,6 +3,11 @@ import { DotCMSHttpClient } from '../utils/DotCMSHttpClient';
 import { DotCMSPageAsset, DotCMSError } from '../models';
 import { DotApiLanguage } from './DotApiLanguage';
 
+export const enum DotCMSPageFormat {
```
Comment (fmontes) [inline projects/dotcms/src/lib/api/DotApiPage.ts:6]: Can you please move this enum to it own file inside: `projects/dotcms/src/lib/models`

Comment (samuelmaddock) [reply projects/dotcms/src/lib/api/DotApiPage.ts]: Done!

File: projects/dotcms/src/lib/api/DotApiPage.ts [hunk 2]
```diff
@@ -32,7 +35,7 @@ export class DotApiPage {
 
         params = {
             ...params,
-            url: `/api/v1/page/json${params.url}`
+            url: `/api/v1/page/${format}${params.url}`
```
Comment (freddyucv) [inline projects/dotcms/src/lib/api/DotApiPage.ts:38]: test?

Comment (samuelmaddock) [reply projects/dotcms/src/lib/api/DotApiPage.ts:38]: Added in 458afa2. However, I wasn't able to actually run the tests. Is it possible to only run the tests for the `dotcms` project?

**Comment by fmontes** [comment]
Thank you @samuelmaddock

**Pull request merged**